### ISSUE TITLE
Unable to build and run

### ISSUE THREAD

**Issue by eleumasc** [wala/WALA#863]
Hi,
I state that I am a beginner with WALA.

This is my environment:
 - Ubuntu Desktop 20.04.2.0 (freshly installed)
 - openjdk 1.8.0_282
 - javac 1.8.0_282
 - gradle 4.4.1
 - Eclipse 2019-12 (4.14.0), with
   - Eclipse PDE (Plug-in Development Environment) latest
   - Buildship Gradle Integration 3.0

First, I have attempted to install WALA by following [this guide](https://github.com/wala/WALA/wiki/Getting-Started), by importing the project (previously cloned from GitHub) with Gradle into Eclipse.
When building, I get about 140 errors, among which:
 - 'source.target/dependency/com.ibm.wala.core.jar' build entry is missing ( from build.properties in /com.ibm.wala.ide )
 - build.properties does not exists ( from com.ibm.wala.core )
 - Package 'com.ibm.wala.analysis.arraybounds.hypergraph.algorithms' does not exist in this plug-in ( from MANIFEST.MF in /com.ibm.wala.ide/META-INF ; this is the most prevalent type of error )

Then, I have noticed that in the guide there is some procedure about Configuring WALA properties, which refers to modify a file called dat/wala.properties ( copied from dat/wala.properties.sample ). Such file has been moved in src/main/resources/ and I don't know if it is still to consider (in every case, the guide on the Wiki is outdated; I think these info were good up to one year ago, by looking at the commit history).

I have also tried to download and execute WALA-Start, always following the guide. It builds successfully with Gradle, but then I don't understand how to run an example (this part is very poorly documented). I have tried to run `./run.py com.ibm.wala.examples.drivers.PDFTypeHierarchy` and `./run.py com.ibm.wala.examples.drivers.ScopeFileCallGraph`, but they both fail with two different kinds of Java's Exception.

So the questions are: How could I make it work? Where am I wrong?

Thank you in advance and I apologize if I seemed a little aggressive, that was not my intention. :D

Samuele

**Comment by msridhar** [comment]
@eleumasc sorry for all the confusion; we really need to clean up the Getting Started guide.

I strongly recommend you follow the "Quick start" instructions at the top of that guide and use WALA-start:

https://github.com/wala/WALA/wiki/Getting-Started#quick-start-using-maven-central-packages

This way you won't have to build WALA from source at all.  You should not need to configure any properties files.

Do you have a specific analysis in mind that you want to run?  If there is an example you'd like to see that is missing from WALA-start, let me know and I can try to add it.

**Comment by eleumasc** [comment]
Thank you for the reply, finally I was able to run some examples in WALA-Start! :)
I report that run.py doesn't work for me (it doesn't pass the arguments to the Java application), so I have updated Gradle to the version 4.9+ and I use the --args parameter, as suggested by [this answer](https://stackoverflow.com/a/29382636).
I have to do some static analysis on Javascript. I have understood how to get the Control Flow Graph of a simple procedure starting from the Call Graph and the IR.
The tested script is this:
`
var i = 5;
var x;
if (i % 2 == 0) x = 3; else x = 2;
`
And the resulting CFG is the following:
![image](https://user-images.githubusercontent.com/29689340/109025147-c374af00-76be-11eb-87f9-85b113d0d834.png)

Now I try something, in case I will write here again.

Thank you,
Samuele
